src/component/WishlistBody.jsx

```diff
@@ -0,0 +1,62 @@
+import { useContext } from "react"
+import { UseWishlistContext } from "../context/wishlist-context";
+import { Rating } from 'react-simple-star-rating'
+
+
+export const WishlistBody= () =>{
+
+const {PostWishlist, wishlist} = UseWishlistContext();
+
+    if(wishlist === "" || wishlist === undefined)
```

Comment:
Suggestion: put the default value of wishlist to "" string. So that you don;t have to check for undefined